Create the wiki for the infrastructure template (#205)
nimblehq/infrastructure-templates: .github/wiki/Contributing-guidelines.md

First off, thank you for considering contributing to this project. It's people like you that make this project such a great tool. We appreciate your help!

The following is a set of guidelines for contributing to this project. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.

## Issues and feature requests

If you encounter any issues or have feature requests, please create an issue on the [GitHub issues](https://github.com/nimblehq/infrastructure-templates/issues) page. Please ensure that a similar issue has not already been created before submitting.

## Development

- Please refer to the [[Architecture Overview]] to understand the project's architecture first. 

> - Then, go through the guidelines under the "Contributing" menu on the right sidebar to set up the development environment, understading the way to create new modules, running tests, and publishing an new release to NPM.

Reviewer: Should it be `Set up the Development Environment` instead?
```suggestion
- Then, go through the guidelines under the "Set up the Development Environment" menu on the right sidebar to set up the development environment, understand how to create new modules, run tests, and publish a new release to NPM.
```

Author: also updated in 4b4c684 & 4f6e753

However, I did not update this part
> go through the guidelines under the "Contributing" menu

to 
> go through the guidelines under the "Set up the Development Environment"

because all of the points (to set up the development environment, understand how to create new modules, run tests, and publish a new release to NPM) are under the "Contributing" menu.

Reviewer: Ah, I misread it 🙈
